src/components/header/controls.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import * as CSSModules from 'react-css-modules';
+import { Bookmark } from './bookmark';
+import * as styles from './controls.scss';
+import { UndoRedo } from './undo-redo';
+
+class ControlsBase extends React.Component<any, any> {
+  public render() {
+    return (
+      <div styleName='controls'>
+        <Bookmark/>
```

Comment:
Let's comment out this line for now so we don't include a button that does not work :)